lib/DMVfacility.rb

```diff
@@ -0,0 +1,30 @@
+class DMVfacility
```

Comment:
Remember to use PascalCase, where we capitalize every new word but keep them together, so `DMVFacility`

Reply:
And then the file name would be `dmv_facility.rb`